crates/astria-sequencer/src/app.rs

```diff
@@ -152,6 +154,8 @@ impl App {
         crate::asset::initialize_native_asset(&genesis_state.native_asset_base_denomination);
         state_tx.put_native_asset_denom(&genesis_state.native_asset_base_denomination);
 
+        // TODO: pass from config
```

Comment:
need to put this in the genesis file, i'll make another PR for this

Reply:
A chain ID is already set in the cometbft genesis?

Reply:
yeah woops i'll update to pass it in from init_chain